src/views/listview/list-view.html

```diff
@@ -1,52 +1,64 @@
-<div class="list-group list-view-pf">
+<div class="list-group list-view-pf" dnd-list="items" ng-class="{'list-view-pf-dnd': config.dragEnabled === true}">
+  <div class='dndPlaceholder'></div>
```

Comment:
If this is a problem, I can style the HTML "li" element to not show up. Just thought this solution was more elegant than trying to hack styles.